clang/include/clang/Frontend/CompileJobCacheResult.h

```diff
@@ -25,6 +25,8 @@ class CompileJobCacheResult : public ObjectProxy {
     MainOutput, ///< Main output file, e.g. object file, pcm file, etc.
     SerializedDiagnostics,
     Dependencies,
+    CASIDFile, ///< A CASID file emitted next to the object file when the
```

Comment:
Total nit: When it doesn't fit on a single line the ///< comments are better replaced with a /// comment on the line above.